packages/shared/components/inputs/RecipientInput.svelte

```diff
@@ -46,28 +47,49 @@
             return Promise.resolve()
         }
 
-        if (value.length !== BECH32_ADDRESS_LENGTH + addressPrefix.length) {
+        if (searchValue.length !== BECH32_ADDRESS_LENGTH + addressPrefix.length) {
             error = localize('error.send.addressLength', {
                 values: {
                     length: BECH32_ADDRESS_LENGTH + addressPrefix.length,
                 },
             })
         } else {
-            error = validateBech32Address(addressPrefix, value)
+            error = validateBech32Address(addressPrefix, searchValue)
         }
 
         if (error) {
             return Promise.reject(error)
         }
         return Promise.resolve()
     }
+
+    $: filteredAccounts = $visibleActiveAccounts?.filter(
+        (account) =>
+            (account.index !== $selectedAccount.index &&
+                account.name.toLowerCase().includes(searchValue?.toLowerCase() ?? '')) ||
+            account.depositAddress.toLowerCase().includes(searchValue?.toLowerCase() ?? '')
+    )
+
+    function getSuffixForAccount(account: IAccountState): string {
```

Comment:
The naming of the function is a bit off here.
```suggestion
    function getTruncatedAddress(account: IAccountState): string {
```